src/main/java/org/apache/sysml/api/DMLScript.java

```diff
@@ -396,7 +568,7 @@ else if (args[i].startsWith("-args") || args[i].startsWith("-nvargs")) {
 				arg.equalsIgnoreCase("-stats") || 
 				arg.equalsIgnoreCase("-exec") ||
 				arg.equalsIgnoreCase("-debug") ||
-				arg.startsWith("-config="))
+				arg.startsWith("-config "))
```

Comment:
I think there is an extra space here: `arg.startsWith("-config "))` --> `arg.startsWith("-config"))`